### ISSUE TITLE
Y24-132 - Security related RT 798566 edit title here Part 2

### ISSUE THREAD

**Issue by TWJW-SANGER** [sanger/sequencescape#4140]
**Describe the Housekeeping**

- [x] Retire old passwords for the **2** user accounts on SequenceScape database mentioned in the ticket. 
- [x] Notify Matt F in case any Tableau reports need updating
- [x] Check deployment project has been update with new secure credentials

Once retired ensure both sets of credentials in the RT no longer authenticate on the retired passwords.

**Blocking issues**
Describe any other issues or tickets that may be blocking this change.

**Additional context**
Details of what to do are here #4043

**Comment by dasunpubudumal** [comment]
Hi @TWJW-SANGER, adding a small note here.

In https://github.com/sanger/sequencescape/issues/4043, I have updated the primary password, retaining the current password for `psdp` user as mentioned in https://github.com/sanger/sequencescape/issues/4043#issuecomment-2061159182. We just need to discard the secondary password using the SQL command:

```sql
ALTER USER 'psdp'@'%' DISCARD OLD PASSWORD;
```

We did not update passwords for the `gapi` user when doing #4043, primarily because we could not identify who that particular user was, and to whom does that user belong.
